Chore/dev override and docs (#10)
nearform/mira: docs/quick-start/README.md

            "account": "YOUR_NUMBER",
            "region": "YOUR_REGION"
          },
          "profile": "YOUR_PROFILE_NAME"
        }]
       ```
    __Note:__ `Default` account is required in the configuration as it is used for personal deployments. See [config documentation](../config/README.md) for more information.

> 6. If you are working in the team, create `config/dev.json` file and provide your own app settings e.g.:

Reviewer: ```suggestion
6. If you are working as part of a team, create a `config/dev.json` file and provide your own app settings e.g.:
```
